src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksperioderMedBegrunnelser/FritekstVedtakbegrunnelser.tsx

```diff
@@ -12,9 +12,7 @@ import { useBehandling } from '../../../../../context/behandlingContext/Behandli
 import { målform } from '../../../../../typer/søknad';
 import type { IFritekstFelt } from '../../../../../utils/fritekstfelter';
 import { hentFrontendFeilmelding } from '../../../../../utils/ressursUtils';
-import HelpText from '../../../../Felleskomponenter/HelpText';
```

Comment:
Dette var en midlertidig lokal klone av designsystemets HelpText med støtte for rikt innhold. Trengs ikke lenger